### ISSUE TITLE
There may be some problems in the positional encoding

### ISSUE THREAD

**Issue by liangzhenlin123** [Kyubyong/transformer#66]
i found u use sum to get the paddind line index in mutihead_attention,however, you may not deal with the padding line in  positional encoding,  which may cause the padding line (After the original absolute value, the sum is 0, and now the sum is not 0)
Below is my modification:
`               position_encoder= tf.convert_to_tensor(position_encoder, tf.float32)
		position_encoder= tf.concat([position_encoder, tf.zeros((1, dims))], 0) 
		position_index_where= tf.sequence_mask(seq, T, tf.bool)
		position_index= tf.where(position_index_where, position_index, tf.ones_like(position_index)* T) 
		output= tf.nn.embedding_lookup(position_encoder, position_index)`

**Comment by liangzhenlin123** [comment]
`		position_encoder= tf.convert_to_tensor(position_encoder, tf.float32)
		position_encoder= tf.concat([position_encoder, tf.zeros((1, dims))], 0) 	
		position_index_where= tf.sequence_mask(seq, T, tf.bool)
		position_index= tf.where(position_index_where, position_index, tf.ones_like(position_index)* T) 
		output= tf.nn.embedding_lookup(position_encoder, position_index)
`
